### ISSUE TITLE
Playerref result show undefined

### ISSUE THREAD

**Issue by Junaid-Amjad** [devcshort/react-hls#28]
Hi, i tried the playerref in my application. but it display undefined in the console. I tried this [example](https://github.com/devcshort/react-hls#using-playerref). Am i missing something? Thanks in advance.

**Comment by devcshort** [comment]
Thanks for bringing this to my attention. I'll need to take a look and test it out. It should be working. Is it possible you're trying to use the ref before it has been assigned?

**Comment by Junaid-Amjad** [comment]
I don't think so. I used the exact example as shown [here](https://github.com/devcshort/react-hls#using-playerref).

**Comment by hoangvu12** [comment]
yea for some reason playerRef return null all the time...
